aiida/cmdline/commands/cmd_import.py

```diff
@@ -78,8 +78,13 @@ class ExtrasImportCode(Enum):
     help="Specify the way to import Comments with identical UUIDs: "
     "newest: Only the newest Comments (based on mtime) (default)."
     "overwrite: Replace existing Comments with those from the import file.")
+@click.option(
+    '--manual-migration', is_flag=True, hidden=True, default=True, help="Do not by default do automatic migration.")
```

Comment:
I'd suggest `--no-migration`.

Also: `default=True`? Why does this even work?
I would also be fine with unhiding this option, since it provides a functionality that is otherwise not available.

Reply:
> I'd suggest `--no-migration` instead.

Yes, but this makes it a bit cryptic to use in the code - but maybe it will be okay?

> Also: `default=True`? Why does this not break the tests?

Yeah, I wondered this myself.

> I would also be fine with unhiding this option, since it provides a functionality that is otherwise not available.

Personally, I think it should be hidden, since under normal circumstances, invoking `verdi import` from the terminal will not do anything - it will wait for the user to say "yes" or "no" to doing the migration. This is only for those cases, where you want to invoke `verdi import` from a Python function, and don't want it to automatically migrate your export files - for one reason or the other.

Reply:
Well, the question about hiding or not just depends on whether you can imagine a user who might want to use this.
E.g. somebody setting up ansible scripts who wants to make sure things are not migrated without notice.
Anyhow, I see no harm in showing that this option exists.

Reply:
Okay. I surrender :) And will also be more strict in forcing this option through in the function as suggested by you in another comment.

Reply:
This discussion is continued in another review comment. Will resolve this one to simplify.